apps/monitoring/docker-compose.yml

```diff
@@ -0,0 +1,71 @@
+version: "3.7"
+
+services:
+  web:
+    image: grafana/grafana:7.5.7
+    user: "1000:1000"
+    restart: on-failure
+    stop_grace_period: 1m
+    links:
+      - prometheus
+    ports:
+      - ${APP_MONITORING_PORT}:3000
+    volumes:
+       - ${APP_DATA_DIR}/data/grafana/data:/var/lib/grafana
+       - ${APP_DATA_DIR}/data/grafana/provisioning:/etc/grafana/provisioning
+    environment:
+      GF_SECURITY_ADMIN_USER: admin
+      GF_SECURITY_ADMIN_PASSWORD: moneyprintergobrrr
+    networks:
+      default:
+        ipv4_address: $APP_MONITORING_IP
+
+  prometheus:
+    image: prom/prometheus:v2.27.1
+    user: "1000:1000"
+    ports:
+      - "9090:9090"
+    volumes:
+      - ${APP_DATA_DIR}/data/prometheus/data:/prometheus
+      - ${APP_DATA_DIR}/data/prometheus/config:/etc/prometheus
+    command:
+      - '--storage.tsdb.path=/prometheus'
+      - '--config.file=/etc/prometheus/prometheus.yml'
+    links:
+      - node_exporter
+      - bitcoin_exporter
+      - lndmon
+    restart: on-failure
+
+  node_exporter:
+    image: prom/node-exporter:v1.1.2
+    volumes:
+      - /proc:/host/proc:ro
+      - /sys:/host/sys:ro
+      - /:/rootfs:ro
```

Comment:
This is a huge security issue, while proc and sys are already not allowed, this is the biggest issue, why is this needed?